cmake/external/helper_functions.cmake

```diff
@@ -134,6 +134,20 @@ macro(onnxruntime_fetchcontent_makeavailable)
       unset(__cmake_haveFpArgs)
     endif()
 
+    # For Project, lik Abseil project, isn't in the external directory.
+    if(MSVC)
+      # Replace /Zi and /ZI with /Z7
```

Comment:
We could also consider using: https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_DEBUG_INFORMATION_FORMAT.html

Reply:
updated.